paper/paper.md

```diff
@@ -75,12 +73,16 @@ Both MACH and MPhys use pyOptSparse [@Wu2020a] to interface with optimization al
 <!-- HMH: I think <both frameworks> here referred to the frameworks pyGeo is directly integrated into -->
 pyGeo passes design variables and constraints to pyOptSparse directly, reducing user effort.
 
+pyGeo was originally developed to implement and use free-form deformation (FFD) to manipulate 3D geometries in CFD-based optimization [@Kenway2010b].
+The "Free-form Deformation" section describes this implementation and recent extensions in more detail. 
+
 pyGeo's interface for design variables and constraints is independent of which discipline solvers are accessing the geometry.
 This means that pyGeo geometries can interact with different types of solvers, such as structures and aerodynamics, in the same way.
 This also allows direct comparison of the behavior or performance of two different solvers within the same discipline using the same geometric parameterization for each, such as two different flow solvers [@Adler2022c].
 
 ## Geometry Generation
-
+<!-- [ ] TODO AY-: I suggest either completely removing this section, or moving it right at the end (maybe after the derivative computation section, i.e. after all geometry parameterization related sections). The main point of pygeo is the parameterization, and I think it may confuse people if we put the surface generation up first. -->
+<!-- [ ] TODO AY-: After going through the paper, I am leaning towards removing this section. The geometry generation was never a critical capability of pygeo, and with our recent efforts, I think that part of the code will soon be removed from the pyGeo repo. To keep things simple and to reduce the word count, I suggest removing this. We can modify pyGeo's readme to mirror that change as well. If you really want to keep it, see my comments above about where to put this section -->
```

Comment:
I am fine with removing this section and rather extend a bit other parts - or possibly add a concluding remark ans suggested by Quim.
We should discuss removing the actual capabilities from pyGeo offline